### PR TITLE
remove duplicates and bogus advice

### LINKED ISSUE #37: Running, Running on Windows
The [Medley repository README](https://github.com/Interlisp/medley#readme) has similar but different material on how to install and get started.

I think we'll have users who aren't familiar with GitHub -- so having the getting-started information in the web site makes more sense. 

This issue is about "consistency if replicated": change the README to point to the web site instructions once they are fixed up.

---

Dependent on #12 .  Once it is completed the README.md and the Medley wiki pages should all point to the [Running on Windows](https://interlisp.org/running/running-on-win/) page.

### PR DESCRIPTION
**Pull request by masinter**
This isn't right either but removes duplicate / overlapping conflicting advice.

### CODE REVIEW

File: content/en/running/_index.md [hunk 1]
```diff
@@ -6,45 +6,17 @@ type: docs
 
 There are different ways of running Medley Interlisp on a wide variety of systems.  This section lays out the basics to help you choose:
 
-* Interlisp Online
-* In a Docker container
+* [Interlisp Online](online)
+  * for most systems; see caveats
+* [In a Docker container](running-with-docker)
+  * currently windows x86_64 only
```
Comment (stumbo) [inline content/en/running/_index.md:12]: Is this true?  Or, is the current documentation Windows specific?

File: content/en/running/online/_index.md [hunk 2]
```diff
@@ -6,13 +6,16 @@ type: docs
...
 
 ### Things to note when running online
 
 * Browser compatibility
 * Watch out for control-character conflicts (control-W always gets me)
 * Security not guaranteed
 * Need a solid net connection to our AWS server (currently in Ohio)
-* Back to time-sharing
-* Maximum load 256/16 24 simultaneous sessions
+* Back to time-sharing: Maximum load ~24 simultaneous sessions
```
Comment (stumbo) [inline content/en/running/online/_index.md:21]: __Back to time-sharing__ maybe drop this, I didn't see any other comments about time-sharing above, so I'm assuming this is just reiterating that there is a maximum number of current users.

File: content/en/running/running-with-docker.md [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
 
-You'll also need a modern VNC client; [TightVNC](https://www.tightvnc.com/) works well. Rumor has it that Macs and Linux use VNC normally for screen sharing.
+When running with Docker, Medley runs in a Docker container using the Interlisp/medley image found on Docker Hub.  A VNC Viewer window is used to display the Medley desktop.  All of this is started up using the single `medley` command.
+
+These instructions are for Windows 10 or 11 (x86_64) only.
+
+
+#### To install and run Medley with Docker Desktop:
+
+1.  *Install Docker Desktop:*&nbsp;&nbsp;Install Docker Desktop for Windows as described [here](https://docs.docker.com/desktop/install/windows-install/).
```
Comment (stumbo) [inline content/en/running/running-with-docker.md:17]: I'd list this as a prerequisite with a link to Docker's site.

**Review by stumbo — CHANGES_REQUESTED** [review]

File: content/en/running/online/_index.md (hunk 2 above)
Comment (masinter) [reply content/en/running/online/_index.md:21]: Interlisp started on time-sharing and it was disruptive to family life -- you had to work later than most to get more cycles. The d-machines were an escape from time-sharing. Running online and we're suddenly returning. i admit this doesn't explain that, but it's also a situation we can quickly overcome if we need to.

File: content/en/running/running-with-docker.md (hunk 3 above)
Comment (masinter) [reply content/en/running/running-with-docker.md:17]: As @fghalasz had it, there was a goal of making "one thing to install", and having that installer install all prerequisites. Docker for Desktop and WSL aren't that way now -- if you don't have them you have to install separately.

File: content/en/running/_index.md (hunk 1 above)
Comment (masinter) [reply content/en/running/_index.md:12]: I think @fghalasz was saying that the _installer_ was windows specific. I'm not sure this wording captures that. As it is, it says "currently" and when that changes, so can this.

**Comment by masinter** [comment]
There are some questions about docker for other platforms, but the instructions here are at least consistent.

**Comment by masinter** [comment]
I think the proposed changes are better than what's there.
I want to point to docker desktop instructions to retain Interlisp's Open Source designation by Docker Hub.

**Pull request merged**